airbyte-workers/src/main/java/io/airbyte/workers/general/DefaultReplicationWorker.java

```diff
@@ -90,11 +97,23 @@ public DefaultReplicationWorker(final String jobId,
     this.messageTracker = messageTracker;
     this.executors = Executors.newFixedThreadPool(2);
     this.recordSchemaValidator = recordSchemaValidator;
+    this.workspaceId = workspaceId;
+    this.dockerImage = dockerImage;
 
     this.cancelled = new AtomicBoolean(false);
     this.hasFailed = new AtomicBoolean(false);
   }
 
+  public DefaultReplicationWorker(final String jobId,
```

Comment:
@lmossman do you know if there have been discussions about moving everything to the container orchestrator?

Reply:
@lmossman nice! this is from 0.39.0-alpha onwards?

Does that main the docker deployment is not running the container orchestrator for now?

Reply:
That's correct - part of the changes the [Migrate OSS to temporal scheduler PR](https://github.com/airbytehq/airbyte/pull/12757) was setting the `CONTAINER_ORCHESTRATOR_ENABLED` env var to `true` for all kube deploys. It is still not set in the docker-compose env file, so it will default to the value of `false` for those deployments